subsystem/kubeapi_test.go

```diff
@@ -1636,7 +1636,7 @@ var _ = Describe("[kube-api]cluster installation", func() {
 				agent.Status.NtpSources[0].SourceName == common.TestNTPSourceSynced.SourceName &&
 				agent.Status.NtpSources[0].SourceState == models.SourceStateUnreachable
 
-		}, "30s", "10s").Should(BeTrue())
+		}, "60s", "10s").Should(BeTrue())
```

Comment:
It's strange that we have a time issue here. i would expect the monitor to run every second so it should catch it fast. 
please decrease the interval to 1 second. no need to wait that long.

Reply:
done